### ISSUE TITLE
Distribute new flyers/posters to places we can advertise

### ISSUE THREAD

**Issue by amcewen** [DoESLiverpool/somebody-should#75]
Once we've got the new flyers/posters designed in issue #32 we should distribute them :-)

This is a place to collect a list of places we'll hit with the flyers, etc. once they're done:
- [ ] Blue Orchid (Neil Morrin can help with that, see #67)
- [ ] LJMU
- [ ] Inform the mailing list(s), so people can collect any relevant materials and stick up in their office, give out to friends, etc.

(When this task gets completed, we should probably copy the final list of locations over to the wiki for future reference)

**Comment by MartinDesigner** [comment]
Try Createer.com. You can easily design online ads, responsive landing pages and print materials such as flyers, posters, covers, invitations and more. Createer lets you easily share your designs on social media, send them via email, download as HTML, image or PDF. Createer also includes 3,000 predesigned template (look here https://beta2.createer.com/Templates) s and over 250,000 images (illustrations, photos and icons) to work with in its user-friendly drag’n’drop enviroment.

**Comment by DoESsean** [comment]
@zarino is happy to design the flyers/posters, but wouldn't be comfortable writing copy.

@johnmckerrell could you liaise with him and get him whatever you have?

**Comment by zarino** [comment]
I'm totally happy to co-write it with someone, if required – I just know I can't handle it all on my own, primarily because I'm not accustomed to the DoES voice and messaging. :-D

**Comment by amcewen** [comment]
We should revisit this once #19 is done, as we can reuse/rework some of the copy from it.

**Comment by DoESsean** [comment]
We are hoping to tie this in with the website meetings so that the message and branding can be uniform cross all media.

**Comment by zarino** [comment]
Closing in favour of the newer https://github.com/DoESLiverpool/somebody-should/issues/1097
